renovate.json

```
@@ -0,0 +1,3 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json"
+}
```

pra gente centralizar a config: https://github.com/quero-edu/renovate
```suggestion
{
  "extends": ["github>quero-edu/renovate:product"]
}

```